compiler/testData/codegen/bytecodeText/intRangeNoBoxing.kt

```diff
@@ -12,7 +12,19 @@ fun foo() {
 // 1 INVOKEVIRTUAL kotlin/ranges/IntRange.getLast \(\)I
 // 0 INVOKEVIRTUAL kotlin/ranges/IntRange.getLast \(\)Ljava/lang/Integer;
 
+// JVM non-IR does NOT specifically handle "step" progressions. The stepped progression in the above code are constructed and its
+// first/last/step properties are retrieved.
+// JVM IR has an optimized handler for "step" progressions and elides the construction of the stepped progressions.
+
+// JVM_TEMPLATES
 // 1 INVOKEVIRTUAL kotlin/ranges/IntProgression.getFirst \(\)I
```

Comment:
Please correct templates to use simple function name for 0 count (like in primitiveProgression.kt):
e.g. 
//0 INVOKEVIRTUAL kotlin/ranges/IntProgression.getFirst \(\)Ljava/lang/Integer; 
->
//0 getFirst

Reply:
It seems that the origin and purpose of the test is to make sure the return types for `getFirst/last/step` are not boxed. I'm splitting this up into 2 test files (range and progression) so that it's clearer.

Reply:
Minor, but I would preferer
```
// 1 INVOKEVIRTUAL kotlin/ranges/IntProgression.getFirst \(\)I
// 1 getFirst
and so on
```
and same in intProgressionNoBoxing.kt

Reply:
@max-kammerer Updated the test